<ui version="4.0" >
 <class>pqLookmarkInspector</class>
 <widget class="QWidget" name="pqLookmarkInspector" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QFrame" name="ButtonFrame" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>2</number>
      </property>
      <property name="spacing" >
       <number>2</number>
      </property>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="LoadButton" >
        <property name="minimumSize" >
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="text" >
         <string>Display</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SaveButton" >
        <property name="minimumSize" >
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="text" >
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="DeleteButton" >
        <property name="text" >
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="PropertiesFrame" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>2</number>
      </property>
      <property name="spacing" >
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="LookmarkIcon" >
        <property name="frameShape" >
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="pixmap" >
         <pixmap/>
        </property>
        <property name="alignment" >
         <set>Qt::AlignCenter</set>
        </property>
        <property name="margin" >
         <number>2</number>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>2</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>1</hsizetype>
            <vsizetype>5</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text" >
           <string>Name: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LookmarkName" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>7</hsizetype>
            <vsizetype>0</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>0</width>
            <height>18</height>
           </size>
          </property>
          <property name="alignment" >
           <set>Qt::AlignLeading</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="pqCollapsedGroup" native="1" name="groupBox" >
        <property name="title" >
         <string>Pipeline View</string>
        </property>
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>2</number>
         </property>
         <property name="spacing" >
          <number>2</number>
         </property>
         <item>
          <widget class="pqFlatTreeView" native="1" name="PipelineView" />
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="pqCollapsedGroup" native="1" name="groupBox_2" >
        <property name="title" >
         <string>Description</string>
        </property>
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>2</number>
         </property>
         <property name="spacing" >
          <number>2</number>
         </property>
         <item>
          <widget class="QTextEdit" name="LookmarkComments" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>5</hsizetype>
             <vsizetype>5</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>0</width>
             <height>18</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="ControlsFrame" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>34</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>2</number>
      </property>
      <property name="spacing" >
       <number>2</number>
      </property>
      <item>
       <widget class="QCheckBox" name="RestoreCamera" >
        <property name="minimumSize" >
         <size>
          <width>0</width>
          <height>18</height>
         </size>
        </property>
        <property name="text" >
         <string>Use Lookmark's camera settings</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="RestoreTime" >
        <property name="text" >
         <string>Use Lookmark's time value</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>308</width>
       <height>51</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pqCollapsedGroup</class>
   <extends>QWidget</extends>
   <header>pqCollapsedGroup.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqFlatTreeView</class>
   <extends>QAbstractScrollArea</extends>
   <header>pqFlatTreeView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QAbstractScrollArea</class>
   <extends>QWidget</extends>
   <header>qabstractscrollarea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../pqComponents.qrc" />
 </resources>
 <connections/>
</ui>
